### PR TITLE
Added a warning to Fruux: proprietary application

### PR DESCRIPTION
**Pull request by TheNoobWar**
## Description

Resolves: Added a warning badge to Fruux under the Worth Mentioning section as it is a proprietary application.

### CODE REVIEW

**Comment by dngray** [comment]
I'm actually wondering if this should even be on PrivacyGuides at all.

From what I can tell it was always there and never open source. It is not encrypted at rest either.

File: _includes/legacy/sections/calendar-contacts-sync.html [hunk 1]
```diff
@@ -55,6 +55,11 @@ <h3>Worth Mentioning</h3>
 <ul>
   <li>
     <a href="https://fruux.com/">fruux</a> - A unified contacts/calendaring system that works across platforms and devices.
+    {% include badge.html
+    color="warning"
+    icon="fas fa-exclamation-triangle"
+    tooltip="The source programming code for Fruux is not available. Fruux is a proprietary application."
```
Comment (Mikaela) [inline _includes/legacy/sections/calendar-contacts-sync.html:61]:
```suggestion
    tooltip="The source code for Fruux is not available. Fruux is a proprietary application."
```

The word "programming" in the middle here looks a bit strange for me

**Review by Mikaela — CHANGES_REQUESTED** [review]

**Review by Mikaela — APPROVED** [review]
Approving for now unless a delisting PR happens.

**Comment by freddy-m** [comment]
> I'm actually wondering if this should even be on PrivacyGuides at all.

@dngray I don't think it should be either.

**Review by Mikaela — APPROVED** [review]
I think it should possibly be in a different PR that says `Resolves: #283` 🤷🏻

**Comment by freddy-m** [comment]
> I think it should possibly be in a different PR that says Resolves: #283 🤷🏻

Nevermind

**Pull request merged**